project_task_pull_request_state/models/project_project.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models, api
+
+
+class ProjectState(models.Model):
+    _inherit = "project.project"
+    _name = "project.project"
+
+    pr_state_default = fields.Selection(
+        selection=lambda self: self.env['project.task']._selection_pr_state(),
```

Comment:
The better solution is to add a function for preparing selection values.
Function name: `_default_pr_state_default`.